### ISSUE TITLE
Snapshots getting wrong items

### ISSUE THREAD

**Issue by carvalho23lucas** [Heishe/idleraiders-second-run#529]
I had 6x Ignites with +50% buff duration
After a while, I decided to switch to +10% Damage. But instead of discarding the previous enchantments, I took another 6x Ignites and enchanted them instead.

But every time I create a new snapshot with the new Ignites equiped, after I use it, it switches back to the old ones.
I've tried deleting all snapshots, putting the new ones and creating again. But the old ones keeps being chosen to be equipped.

**Comment by askebv** [comment]
Yeah, I've had the same issue for a while. As a temporary fix, I've overwritten or extracted all my worse enchants and only went with one enchant style per skill. Still hope it gets fixed at some point.
